includes/emails/class-wcs-email-customer-notification.php

```diff
@@ -0,0 +1,139 @@
+		$this->form_fields = array(
+			'enabled'            => array(
+				'title'   => __( 'Enable/Disable', 'woocommerce-subscriptions' ),
+				'type'    => 'checkbox',
+				'label'   => __( 'Enable this email notification. Disabled automatically on staging sites.', 'woocommerce-subscriptions' ),
+				'default' => 'yes',
+			),
+			'subject'            => array(
+				'title'       => __( 'Subject', 'woocommerce-subscriptions' ),
+				'type'        => 'text',
+				'desc_tip'    => true,
+				'description' => $placeholder_text,
+				'placeholder' => $this->get_default_subject(),
+				'default'     => '',
+			),
+			'heading'            => array(
+				'title'       => __( 'Email heading', 'woocommerce-subscriptions' ),
+				'type'        => 'text',
+				'desc_tip'    => true,
+				'description' => $placeholder_text,
+				'placeholder' => $this->get_default_heading(),
+				'default'     => '',
+			),
+			'additional_content' => array(
+				'title'       => __( 'Additional content', 'woocommerce-subscriptions' ),
+				'description' => __( 'Text to appear below the main email content.', 'woocommerce-subscriptions' ) . ' ' . $placeholder_text,
+				'css'         => 'width:400px; height: 75px;',
+				'placeholder' => __( 'N/A', 'woocommerce-subscriptions' ),
+				'type'        => 'textarea',
+				'default'     => $this->get_default_additional_content(),
+				'desc_tip'    => true,
+			),
+			'email_type'         => array(
+				'title'       => __( 'Email type', 'woocommerce-subscriptions' ),
+				'type'        => 'select',
+				'description' => __( 'Choose which format of email to send.', 'woocommerce-subscriptions' ),
+				'default'     => 'html',
+				'class'       => 'email_type wc-enhanced-select',
+				'options'     => $this->get_email_type_options(),
+				'desc_tip'    => true,
+			),
+		);
+	}
+
+
+	/**
+	 * Trigger function.
+	 *
+	 * @return void
+	 */
+	public function trigger( $subscription_id ) {
+		$subscription    = wcs_get_subscription( $subscription_id );
+		$this->object    = $subscription;
+		$this->recipient = $subscription->get_billing_email();
+
+		if ( ! $this->is_enabled()
+			|| ! $this->get_recipient()
+			|| ! WC_Subscriptions_Email_Notifications::should_send_notification()
+			|| WC_Subscriptions_Email_Notifications::subscription_period_too_short( $subscription )
+		) {
+			return;
+		}
+
+		$result = $this->send( $this->get_recipient(), $this->get_subject(), $this->get_content(), $this->get_headers(), $this->get_attachments() );
+
+		if ( $result ) {
+			/* translators: 1: Notification type, 2: customer's email. */
+			$order_note_msg = sprintf( __( '%1$s was successfully sent to %2$s.', 'woocommerce-subscriptions' ), $this->title, $this->recipient );
+		} else {
+			/* translators: 1: Notification type, 2: customer's email. */
+			$order_note_msg = sprintf( __( 'Attempt to send %1$s to %2$s failed successfully.', 'woocommerce-subscriptions' ), $this->title, $this->recipient );
```

Comment:
```suggestion
			$order_note_msg = sprintf( __( 'Attempt to send %1$s to %2$s failed.', 'woocommerce-subscriptions' ), $this->title, $this->recipient );
```

A successful failure. 🤔

Reply:
Just a reference to the famous Windows message, as I knew we would be replacing the placeholder texts with copy from Studio.
![iKzHLJY](https://github.com/user-attachments/assets/310c1728-c95a-4083-ac11-aef61f532c32)